assembly/src/main/dist/docker/build.sh

```diff
@@ -14,10 +14,10 @@ rm -rf ../../../../target/
 
 # go to root of project and do clean, format, install and assembly
 cd ../../../../..
-MVN_VERSION=$(mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
+MVN_VERSION=$(xmllint --xpath "//*[local-name()='project']/*[local-name()='version']/text()" pom.xml)
+
 mvn clean
-mvn formatter:format
-mvn -Passembly install -DskipTests
+mvn -T 2 -Passembly install -DskipTests -Dmaven.javadoc.skip=true -Dmaven.test.skip=true -Dformatter.skip=true -Dimpsort.skip=true -Dxml-format.skip=true  -Djapicmp.skip -Denforcer.skip=true -Dbuildnumber.plugin.phase=none -Danimal.sniffer.skip=true
```

Comment:
I'm not entirely sure why but this command failed for me on the first run, reporting that it couldn't find the rdf4j-workbench.war (as part of the assembly process). On a second run (after removing the `-T 2`) it succeeded, but I'm not sure if that is because I removed the parallel execution flag or because I just ran it a second time.